strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/GroupRepositoryProvider.java

```diff
@@ -58,6 +59,9 @@
     @Inject
     private HostedRepositoryProvider hostedRepositoryProvider;
 
+    @Inject
+    private GroupRepositorySetCollector groupRepositorySetCollector;
```

Comment:
where do this changes came from?

Reply:
From the rebase ! Rebasing other PRs into my branch !

Reply:
your commits should be above the master head in case of rebase, so we shouldn't see them here.

Reply:
I mean that we should see only your commits after rebase, that's all the point of rebase :)

Reply:
also you should rebase on master, but not on other PRs